### DISCUSSION TITLE
Change top notification wording

### DISCUSSION THREAD

**Discussion by renueman** [Ylianst/MeshCentral#4575] [General]
Is there a way to change the wording in the notification bar at the top of the client screen from "... has started a remote session" to something else?

Possibly a setting in the config file?

**Comment by krayon007** [comment]
I think one way you can accomplish that is to modify the translation json file

**Reply by renueman** [reply]
Good deal. Now, I just need to find the file (??). Where might that be?

**Reply by krayon007** [reply]
> Good deal. Now, I just need to find the file (??). Where might that be?

It's in the translate folder of your server. Refer to the readme in that folder for help. 
But if you search the translate.json file, look for the string: `Sharing desktop with: {0}`
You can reword that however you like. The value `{0}` will get replaced with the username, so be sure to incorporate that with whatever you reword it to.

**Reply by renueman** [reply]
"Sharing desktop with" I changed it to "{0} , Has logged into your computer for remote support",
Restarted meshcentral and not seeing change.
Now, I am running linux and also assuming that downloading the windows program to run on translate.json is to update the other languages in the file (?). I figure I do not need to do that as I only use English.

**Comment by krayon007** [comment]
I didn't see the string `has started a remote session`. Is that what your notification bar at the top of the screen says?

**Reply by renueman** [reply]
It says "Sharing desktop with <myfirstname>" in the top banner.

**Comment by renueman** [comment]
Perhaps the wording change in the translate.json file not showing is because of a cache still using the unedited data or the agent on the guest computer needs to be updated - something like that. Ill have to find time to look into this.
